packages/components/text/src/Text.tsx

```diff
@@ -52,6 +53,16 @@ export const Text = compressible<TextProps, TextTokens>((props: TextProps, useTo
 
   /* Only if onPress is set */
   const pressable = useAsPressable({ ...rest, onPress });
+  const textOnPress = React.useCallback(
+    (e) => {
+      if (onPress) {
+        onPress(e);
+      }
+      e.stopPropagation();
+    },
+    [onPress],
+  );
+  const onKeyUpProps = useKeyProps(textOnPress, ' ', 'Enter');
```

Comment:
nit: This hook returns `keyDown` for macOS, and `keyUp` for windows, since I found that to be a platform difference which was the default. So a better name would exclude the "up" to avoid confusion